File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getBlockReceipts.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");

(async () => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockHash =
    "0xba647d41423faeebe8a7c64737d284fc2eba6f0388a3e1ebf6243db509ec1577";

  // The full list of JSON-RPC is available at:
  // https://docs.klaytn.foundation/content/dapp/json-rpc/api-references
  const data = await provider.eth.getBlockReceipts(blockHash);
  console.log("Block receipts", data);
})();

File: web3rpc/rpc-specs/code-samples/python/eth/block/getBlockReceipts.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xba647d41423faeebe8a7c64737d284fc2eba6f0388a3e1ebf6243db509ec1577"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_block_receipts(blockHash)

print(eth_response)
